install/helm/kgateway/templates/deployment.yaml

```
@@ -77,6 +75,12 @@ spec:
               value: {{ .Values.image.tag | default .Chart.Version }}
             - name: KGW_DEFAULT_IMAGE_PULL_POLICY
               value: {{ .Values.image.pullPolicy | default "IfNotPresent" }}
+            {{- if .Values.controller.extraEnv }}
+            {{- range $key, $value := .Values.controller.extraEnv }}
+            - name: {{ $key }}
+              value: {{ $value | quote }}
+            {{- end }}
+            {{- end }}
             # TODO: Remove this once the cleanup is done. Required as the gloo-system
```

What's the default kgateway namespace? kgateway-system? 
```suggestion
            # TODO: Remove this once the cleanup is done. Required as the kgateway-system
```

---

Yeah technically it's `kgateway-system` but I'm not really certain the state of this env var this TODO is referring to. I'll scrub the `gloo-system` reference though

---

Actually on a second look I'm not sure the implications here, I'll leave as-is for now if that's cool